packages/arb-token-bridge-ui/src/state/app/actions.ts

```diff
@@ -59,13 +57,6 @@ export const setWarningTokens = (
   state.app.warningTokens = warningTokens
 }
 
-export const setWhitelistState = (
```

Comment:
How come all the whitelist stuff is removed?

Reply:
It wasn't used anywhere, but we can do that in a different PR

Reply:
I think it should be fine here